kroxylicious-filter-test-support/src/main/java/io/kroxylicious/test/record/RecordTestUtils.java

```diff
@@ -226,24 +227,70 @@ public static Record record(byte magic,
         return MemoryRecords.readableRecords(mr.buffer()).records().iterator().next();
     }
 
+    /**
+     * Return a singleton RecordBatch containing a single Record with the given key, value and headers.
+     * The batch will use the current magic. The baseOffset and offset of the record will be 0
+     * @see RecordTestUtils#recordBatch(long, String, String, Header[])
+     */
+    public static MutableRecordBatch recordBatch(String key,
```

Comment:
could we give this method a name which describes the fact that the batch contains a single record?